README.md

```diff
@@ -3,26 +3,111 @@
 ![Build Status](https://github.com/CommerceWeavers/SyliusSaferpayPlugin/actions/workflows/build.yaml/badge.svg)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-This plugin adds integration with [Wordline Saferpay](https://www.saferpay.com) to [Sylius](https://github.com/Sylius/Sylius). 
+![Sylius](docs/img/sylius.png) ![Wordline](docs/img/wordline.png)
 
 ## Content
 
-1. [Requirements](docs/requirements.md)
+1. [Requirements](#requirements)
+1. [Overview](#overview)
+1. [Features](#features)
 1. [Installation](docs/installation.md)
 1. [Configuration](docs/configuration.md)
 1. [Development](docs/development.md)
 1. [Testing](docs/testing.md)
+1. [Cooperation](#cooperation)
+1. [License](#license)
+1. [About us](#about-us)
+
+### Requirements
+
+| Package       | Version |
+|---------------|---------|
+| PHP           | ^8.0    |
+| sylius/sylius | ^1.10.0 |
+
+### Overview
+
+This plugin adds integration with [Wordline Saferpay](https://www.saferpay.com) to [Sylius](https://github.com/Sylius/Sylius).
+
+#### About Saferpay
+
+Saferpay is a modern, all-in-one payment solution, particularly popular in the DACH region due to origins of its
+previous owner - SIX Payments - which is a Swiss fintech acquired by French Wordline group. As part of Wordline group,
```

Comment:
Please change the sentence to: a Swiss company acquired by Worldline - a French fintech group. 

And delete the beginning of next sentence: ~As part of Wordine group,~